app/src/main/java/com/kickstarter/viewmodels/DiscoveryViewModel.java

```diff
@@ -223,7 +77,7 @@ public DiscoveryViewModel(final @NonNull Environment environment) {
       .mergeWith(topFilterRowClick)
       .map(NavigationDrawerData.Section.Row::params);
 
-    // Merge various param data sources to one true selected params value.
```

Comment:
> one true selected params value

lol at myself because. We didn't need `params`, `paramsOnSortChange`, and `paramsOnDrawerSelection`. In fact, having those three separate `params` observables caused some values to over-emit.

We just need `params` to set DiscoveryActivity UI, and the `params` merged with sort position for the fragments.